### PR TITLE
[OSDEV-962] Update release protocol

### PR DESCRIPTION
**Pull request by Innavin369**
[OSDEV-962](https://opensupplyhub.atlassian.net/browse/OSDEV-962) Update release protocol

The Release protocol has been updated after the automatization of manual processes such as creating a release branch, restoring DB, deploy to AWS.

### CODE REVIEW

File: doc/release/RELEASE-PROTOCOL.md [hunk 1]
```diff
@@ -131,16 +131,24 @@ Make sure that:
...
+9. As the final step, shut down the pre-prod environment, as it is no longer required for the current version of the release.
 
 ### Hotfixes
 
 - To deploy a hotfix to pre-prod, you should fork from the latest release branch, and after preparing the fix, merge it back. Merging will trigger the Deploy to AWS workflow that will deploy the hotfix to the **running** pre-prod environment. Before doing it, don't forget about the step involving switching databases, as mentioned in [the Code Freeze section]((#requirements-and-key-results)).
 - To release a hotfix to production and staging, you should fork from the latest release branch, and after preparing the fix, merge it back. The last step is to execute the Release [Deploy] workflow for each environment separately, which will deploy the fix to these two environments.
+
+### Shut down the pre-prod environment
+
+- To shut down the Pre-prod the responsible personhave to select `Destroy Environment` in the Actions menu. Press run worflow with `main` branch and select `Pre-prod` env.
```
Comment (VadimKovalenkoSNF) [inline doc/release/RELEASE-PROTOCOL.md:153]: Add whitespace: `person have to ...`

**Review by VadimKovalenkoSNF — COMMENTED** [review]
Add minor comment

File: doc/release/RELEASE-PROTOCOL.md [hunk 2]
```diff
@@ -131,16 +131,24 @@ Make sure that:
...
+7. Notify the QA Engineer that the new version has been released, and they can commence testing the items listed under the *prod* column in [the QA Checklist sheet](https://docs.google.com/spreadsheets/d/
+8. Upon completing the release, the responsible person must notify stakeholders in the *#data_x_product* Slack channel that the releases to sandbox and production have concluded. Additionally, update the *Unreleased* version's status in Jira.
+1uinHJOPpGfrUNkewBxPVsDeDXnNx4dJnnX94LoBA_zk/edit?usp=sharing).
```
Comment (vladsha-dev) [inline doc/release/RELEASE-PROTOCOL.md:143]: What is this? Did you want to add a link?

**Review by vladsha-dev — CHANGES_REQUESTED** [review]
Left comments

File: doc/release/RELEASE-PROTOCOL.md (hunk 2 above)
Comment (Innavin369) [reply doc/release/RELEASE-PROTOCOL.md:143]: That was the link from the previous option. Fixed.

**Review by VadimKovalenkoSNF — APPROVED** [review]
Approved

**Review by vladsha-dev — APPROVED** [review]
LGTM

**Comment by Innavin369** [comment]
PR was closed due to branch corruption. PR 141 has been created insted.

**Pull request closed without merging**